### ISSUE TITLE
member.site * Under Construction *

### ISSUE THREAD

**Issue by kitblake** [rchain/bounties#595]
# member.site * Under Construction *

Specific: we have a Wordpress instance running for content input.
Measurable: the goal is to have the MVP site structure realized as defined in:
  https://docs.google.com/document/d/1JXM8k-fzf78Y_GbfkDdsekmdeVqqifC4VM0zEMBDxDw/edit
Achievable: a few activists are working on it, but design and content input are needed.
Realistic: end of the month is an appropriate deadline, but the KYC app is an unknown.
Timely: improving onboarding is a top priority. 

Measure of Completion:  MVP site is live
Suggested Budget: $5000 (125 collective hours)

**Comment by kitblake** [comment]
## Status
Last weekend @dckc and friends got the dev Wordpress site running and configured with a default theme. It's not lovely but perfectly suitable for inputting content. Design mockups can happen in parallel. 

There's a YouTube portlet plugin that works great. It pops up an overlay, and that keeps the user in the site. √

Recent changes:
- Nav bar menu configured with the top level links in order
- Fine tuned the sidebar, removed useless elements
- Testing having nav in the sidebar as opposed to a header bar, need another sidebar
- Setup the Support page with content
- Added social media icons menu
- Configured app icon (favicon)

**Comment by Barkov-F** [comment]
What multilanguage support do you plan to have? What do you think of [Polylang](https://wordpress.org/plugins/polylang/) plugin?

**Comment by kitblake** [comment]
Hi @Barkov-F, at this point we're working on a Minimal Viable Product of the member.site which doesn't include multi-language support. 

Personally I don't know anything about the Polylang plugin. Their issue tracker does not inspire confidence:
https://wordpress.org/support/plugin/polylang

**Comment by kitblake** [comment]
### Calendar
One of the key aspects of the member.site is the Calendar. I'm sure we can find a plugin that'll embed a remote (Google) calendar. 

The question could be: which existing calendar do we want to embed? @patrick727?

Or the question is: shall we create a calendar especially for the member.site? It could subscribe to existing calendars and list all events that potential Members might find interesting (e.g. working group meetings). In which case we should prob spin off this issue into its own.

**Comment by dckc** [comment]
Somebody tell me a story about the calendar? What does the world look like when it's working?

Bonus points for telling the side of the story where material gets added to the calendar as well as when the calendar is read.

I was about to get into technical stuff (i.e. I don't think one Google calendar can subscribe to others) but without some sense of use cases / requirements, there's no way to judge.

**Comment by dckc** [comment]
As of March 24, we were using [Pats take on member.rchain.coop](https://docs.google.com/presentation/d/1FebyTar-cvUCEgmF5u7F5kUwo55oPHCXQ3svDVMqAI8/edit?usp=sharing) slides to define the MVP. I hope the difference doesn't matter, but I did make a point to be sure all my issues were noted there.

**Comment by kitblake** [comment]
Some progress:
- We have a site hierarchy created that follows the MVP site map. 
- The “Get Involved” section is filled in, including embedded video.

**Comment by kitblake** [comment]
Please welcome Maarten van Domburg, aka @Andrekuipers on Github. Maarten is an experienced web developer with Wordpress knowledge and will be pitching in on the setup and theming of the Member site.

**Comment by kitblake** [comment]
In the RAM meeting I got access to a 'main' (for lack of a better word) calendar, so I'll integrate it into the Calendar page. 

We also discussed getting the About section fleshed out. For nav purposes we may not want it go too deep, so we might make each sub-page a sort of SPA that contains all the content in sections.

**Comment by kitblake** [comment]
We should continue to develop content in the https://coop-member-v1.rchain.me/ site, @Ojimadu (and anybody else). The content will be selectively migrated to the theming dev site. #615

**Comment by kitblake** [comment]
The top page of About has content now: https://coop-member-v1.rchain.me/?page_id=48
Critique welcome, underlying linked pages not yet.

**Comment by kitblake** [comment]
@Ojimadu Go ahead and merge.

**Comment by kitblake** [comment]
Status on the 1st of May: we have draft content (i.e. ready for review/improvements) on Home and these top level pages: About, Get Involved. Membership, Support. ToDo is the top level Calendar page which requires a plugin to embed a Google Cal.

Still to be created, both content and layout, are the subpages in About. These subpages will serve as a template for future subpages in Membership, etc.

On weds May 2nd we'll move everything from the dev instance to the rchain.me instance (#615).

**Comment by kitblake** [comment]
Today I polished the upper level content, subpages of About are still ToDo.

**Comment by dckc** [comment]
I'm struggling to make sense of this:
> The RChain platform is powered by a Turing complete, byzantine fault-tolerant Rho Virtual Machine

help? I thought the BFT related stuff was in a protocol layer above the VM (casper).

**Comment by kitblake** [comment]
In #615 @dckc wrote:
> In our March 24 MVP, I don't see anything regarding subpages for About. submenus makes me nervous... I guess I should look into this further.

Don't worry about the submenus, they're in stock. WordPress is the Ikea of the web.

@pmoorman @dckc The MVP site structure we're following is in:
https://docs.google.com/document/d/1JXM8k-fzf78Y_GbfkDdsekmdeVqqifC4VM0zEMBDxDw/edit
Scroll to the end for outline # 3. Items in gray don't have content ready and are not in the MVP.

**Comment by kitblake** [comment]
Anybody have any content for the Member site? 

We're almost done and will request that the site gets reviewed before launch, so if you have anything (to add, or to improve) please get on it.

**Comment by pmoorman** [comment]
I worked with @kitblake and @Andrekuipers on content today. The MVP is release-ready from a content perspective, but blocked on a few other fronts.

@kitblake will take the lead on resolving subdomain structure / DNS control, and will sync with Patrick about approval & release.  (did I miss anything, Kit?)

For now, no new content is needed... although once we're live there are many things that could be improved upon or potentially even tested for better conversions.

**Comment by pmoorman** [comment]
Few points that I've made in private chat with @kitblake that are maybe worth recording here (or we might forget):

1. 

> why do we use all those different CTAs throughout the pages?  Sometimes we say "join the coop" then "get involved" then "I'd like to join" and so on.
> 
> Shouldn't we stick with a consistent "become a member" (because that's really the next step)?

2. 
> Of course we also have secondary CTAs like "contact us" or "join the discord", but that's a good idea. Shouldn't we make "contact us" point to Discord more aggressively? 
> 
> I wonder if the "contact us" link to a support page is going to convert anyone

3. I have another altogether crazy idea (but here we're getting squarely into marketing territory): I wonder how a single long-form page would convert against an entire website. I wouldn't be too surprised if a single long form "sales page"  (for lack of a better word) would out-pull this configuration. But this is something to test once we 1) have an MVP up and running and 2) agree on what we'd measure as a success KPI

4. Another idea that I think would work well on the member website are CTAs along the line of "talk to the team" and stuff of that sort. In my opinion RChain is differentiated against other projects for it's openness and inclusiveness: you can easily hang out here in Discord and talk directly to Nash, Medha, Ian and other "key players" whereas in other crypto projects you probably wouldn't have such opportunities.

**Comment by Andrekuipers** [comment]
Great!

I have thought about the first point. Because it is more of a kind of footer that comes back on every page I would keep the text the same everywhere. I think it creates more peace and unity for the user.

Point 4 sounds as a really good selling point for RChain. I think it is a 'must' to show this more on the website. People will trust it more quickly and feel more welcome :)

**Comment by dckc** [comment]
@pmoorman writes:
> ... will sync with Patrick about approval & release.

Recall it's @ian-bloom who the executive committee said we need to coordinate with.

**Comment by kitblake** [comment]
For the record, the company descriptions on the Ecosystem page have been approved, by @llerner and GregH and @ leaf/Nash.

**Comment by pmoorman** [comment]
@kitblake see Dan's comment above. You check with either Patrick or Ian (or both), right?

...

@Andrekuipers nice let's go for a unified 'footer' then, if we both think that makes sense. What I've said to Kit before (and I still feel the same) is that these changes are more like 'nice to have' than essential to the MVP. Same for idea #4 which I wouldn't classify as a "must" either. This shouldn't delay us in getting the MVP live.

**Comment by Andrekuipers** [comment]
@pmoorman Oke cool!

Before we migrate the website and going live I only need to check if everything is working responsive. I will do that today.

**Comment by kitblake** [comment]
@pmoorman Yes, Patrick is already teaming up for review. 
I'll be contacting @ian-bloom for the subdomain, but before any infrastructure changes, we want a signoff on the content. Otherwise this could slide into summertime.

**Comment by kitblake** [comment]
I added a "Talk to the team" portlet in: https://coop-member-v1.rchain.me/about/governance/ 
Great idea @pmoorman. We can put it elsewhere too..

**Comment by kitblake** [comment]
⚠️ Last call for content review from the team. 
If no revisions come in, tomorrow we'll submit the MVP for review.

**Comment by dckc** [comment]
Submit to whom? We're all here, no?

And the content will remain editable after putting it at members.rchain.coop,
yes?

**Comment by kitblake** [comment]
@dckc Yes and yes. 

@PatrickM727 @pmoorman @Ojimadu @Andrekuipers @Barkov-F @ian-bloom @evanjensen
The Member site MVP is ready for review: https://coop-member-v1.rchain.me/

**Comment by kitblake** [comment]
The Member Site has passed the review with no changes. 👍 

We'd like to point the domain member.rchain.coop at what is now https://coop-member-v1.rchain.me/

In order to do that the member registration app/site will need to move to a new subdomain. The best suggestion so far is join.rchain.coop. Any better ideas? 

@ian-bloom, can you guys implement these changes? 
Please let us know and we'll adjust any links accordingly before the move..

**Comment by Ojimadu** [comment]
+1 For join.rchain.coop

**Comment by Andrekuipers** [comment]
Good news :)!
+1  join.rchain.coop

**Comment by kitblake** [comment]
This issue is blocked. 12 days ago I posted, asking Ian to make the subdomain changes. This week I asked him about it in a meeting and he said he doesn't read his Github email (use normal email or Discord to contact him). Ian said that @n10n has the keys for the member.rchain.coop subdomain. So I mailed Navneet  butI haven't heard back from him either. Somebody said he's on vacation..

**Comment by dckc** [comment]
In the July 17 [executive committee](https://www.gov.rchain.coop/executive-committee) meeting, @kitblake and I learned that @ian-bloom is standing by for approval from @PatrickM727 :
> IB: Patrick needs to give approval ...

**Comment by dckc** [comment]
Which improvements? What was the date of the board meeting where that was decided?

Or do you mean "members of the board"?

**Comment by kitblake** [comment]
I'm paraphrasing what @PatrickM727 told me, so the answer to "which, what, and who" is idk. It's all ripe for conjecture, but I'll leave that unsaid in order to focus.

The Member site is meant to be "a gateway to getting involved" with the Cooperative that guides potential new members to join.

It's not a gateway to getting involved with the Bounty System. When constructing the site I thought it was useful to include a guide to joining the Bounty System as it's a complex process. But if 'management' is uneasy with that, we can let the instructions live somewhere else until the Bounty System proves its viability to its detractors' appeasement.

I'm thinking of a Members page with info about:
- Annual Members Meeting
- Events - upcoming RCon3 in Berlin
- Regular Meetings
- Regional Meetups
- Bounties info
